go.mod

```diff
@@ -65,8 +65,13 @@ require (
 // When updating cloud.google.com/go/auth, ensure the auth.IsCredentialsNotFoundError still works.
 require cloud.google.com/go/auth v0.12.0
 
+require cloud.google.com/go/logging v1.11.0
```

Comment:
move this into the main require block.

i think the auth one out them explicitly messes up the go.mod handling when you do a go get.